README.md

```diff
@@ -98,17 +97,18 @@ When it terminates, the following command can be used to get access to
 the Kubernetes cluster that was brought up for testing (assuming that
 this step succeeded):
 
-    export KUBECONFIG="$(kind get kubeconfig-path --name="csi-prow")"
+`export KUBECONFIG="$(kind get kubeconfig-path --name="csi-prow")"`
```

Comment:
Three backticks with `console` as type is probably better here.

Reply:
done.